packages/builder/src/internal/core/runtime-environment.ts

```diff
@@ -58,37 +50,17 @@ export class Environment implements BuilderRuntimeEnvironment {
   ) {
     log("Creating BuilderRuntimeEnvironment");
 
-    const networkName =
-      builderArguments.network !== undefined
-        ? builderArguments.network
-        : config.defaultNetwork;
-
-    const networkConfig = config.networks[networkName];
+    const networkConfig = config.networks[builderArguments.network];
 
     if (networkConfig === undefined) {
       throw new BuilderError(ERRORS.NETWORK.CONFIG_NOT_FOUND, {
-        network: networkName,
+        network: builderArguments.network,
       });
     }
 
     log("Runtime environment NOOP (MM)");
-    //const provider = lazyObject(() => {
-    //  log(`Creating provider for network ${networkName}`);
-    //  return createProvider(
-    //    networkName,
-    //    networkConfig,
-    //    config.solc.version,
-    //    config.paths
-    //  );
-    //});
-
-    //this.network = {
-    //  name: networkName,
-    //  config: config.networks[networkName],
-    //  provider,
-    //};
-
-    //this.ethereum = provider;
+    // this.provider .... -- creation goes here
```

Comment:
I left the comments because it wasn't evident what creation was here. At least we have history.